### DISCUSSION TITLE
Admin Password Reset

### DISCUSSION THREAD

**Discussion by Cory74** [netbox-community/netbox#10477] [Q&A] [answered]
I have somehow changed the default admin password and now I can't access netbox.  Is there a way to reset the admin password?  

I am running on Ubuntu as a docker container

**Comment by Cory74** [comment, marked as answer]
Please ignore i have found the correct password

**Reply by candlerb** [reply]
For future reference, [manage.py createsuperuser](https://docs.netbox.dev/en/stable/installation/3-netbox/#create-a-super-user) can be used to create a new admin login.
